fili-core/src/main/java/com/yahoo/bard/webservice/data/dimension/impl/LuceneSearchProvider.java

```diff
@@ -365,9 +365,7 @@ private void refreshCardinality() {
             Set<ApiFilter> filters,
             PaginationParameters paginationParameters
     ) {
-        int perPage = paginationParameters.getPerPage();
-        validatePerPage(perPage);
```

Comment:
I don't think we should lose the validation step. I realize that we're not using `perPage` directly, but we still want to validate it.